darts/tests/dataprocessing/transformers/test_mappers.py

```diff
@@ -65,6 +66,17 @@ def test_mapper_with_timestamp(self):
 
         for to_transform, expected_output in test_cases:
             transformed = self.subtract_month.transform(to_transform)
+            if isinstance(to_transform, list):
+                tmp_t = []
```

Comment:
It looks like `tmp_t == transformed` no? And also that you could build `expected_output` with something like
```
expected_output = [o.with_column_renamed(o.components[0], t.components[0]) for (o, t) in zip(transformed, expected_output)]
```

Reply:
yes indeed, I'll correct it